tomlkit/api.py

```diff
@@ -22,10 +23,10 @@
 from .items import Whitespace
 from .items import item
 from .parser import Parser
-from .toml_document import TOMLDocument as _TOMLDocument
```

Comment:
The type hints of public functions (e.g. loads/dumps) where exposing something that looked a private class...
This makes it complicated to use the library in a typechecked project.